language/stdlib/transaction_scripts/add_currency.move

```diff
@@ -0,0 +1,20 @@
+fun main<NewCurrency>(
+    exchange_rate_denom: u64,
+    exchange_rate_num: u64,
+    is_synthetic: bool,
+    scaling_factor: u64,
+    fractional_part: u64,
+    ticker_symbol: vector<u8>,
+) {
+    let rate = 0x0::FixedPoint32::create_from_rational(
```

Comment:
Nit: the other scripts and modules use the `use 0x0::FixedPoint` style everywhere, so might be good to switch for consistency.